com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/context.py

```diff
@@ -113,9 +113,10 @@ def submit(self):
         # Otherwise, use the Java version from the streams install
         else:
             jvm = os.path.join(streams_install, "java", "jre", "bin", "java")
-            if config[ConfigParams.FORCE_REMOTE] == True:
+            if self.config[ConfigParams.FORCE_REMOTE] == True:
```

Comment:
Is this being treated differently to other configuration parameters, such as `topology.keepArtifacts`?

And can the name be `topology.forceRemoteBuild` as that is what is being forced, a build.